### PR TITLE
To-do-list: Interactive list

### PR DESCRIPTION
**Pull request by V0pe**
Project 3: Interactive list

In this project , I implemented the following to update the interactive apps:
- [ ] Add the completed change upon action.
- [ ] Add the styles for the completed feature selection.
- [ ] Add the clear all completed action.
- [ ] Update the index upon the completed action.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -102,6 +102,33 @@ class Engagement {
...
+          clearall.addEventListener('click', () => {
+            this.storage = this.storage.filter((check) => check.completed === false);
+            localStorage.setItem('tasks-storage', JSON.stringify(this.storage));
+            const todoList = document.querySelector('.todo-list');
+            todoList.innerHTML = '';
+            window.location.reload();
+          });
+        }
+      }
+    }
```
Comment (mmsesay) [inline src/app.js]: - [ ] I noticed something. The checkbox is not updating the upon user actions on the second click. On the first click, the object's status gets updated to `true` and the task gets struck out. When I click for the second time to update the status, from `true` to `false` it doesn't update. Please see the demo gif to get some understanding of what I am pointing out here.

Demo |
:-------:|
a1

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @V0pe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/app.js (hunk 1 above)
Comment (V0pe) [reply src/app.js]: @mmsesay Yes, that was insightful. Apparently, I didn't check for the uncheck box. I have fixed it now

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey  @V0pe 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**